gluten-celeborn/clickhouse/src/main/scala/org/apache/spark/shuffle/CHCelebornColumnarShuffleWriter.scala

```diff
@@ -71,7 +71,12 @@ class CHCelebornColumnarShuffleWriter[K, V](
       }
     }
 
-    assert(nativeShuffleWriter != -1L)
+    // If all of the ColumnarBatch have empty rows, the nativeShuffleWriter still equals -1
```

Comment:
When `!records.hasNext=true`, does this also need to invoke `handleEmptyIterator`?

Reply:
Yes，the logic of invoking `handleEmptyIterator` when `!records.hasNext=true` is in the `wtite` method of super class `CelebornColumnarShuffleWriter`.